src/Form/Form.js

```diff
@@ -65,6 +88,10 @@ class Form {
             this.execOnInputs(this.removeInputDecoration)
             this.listeners.forEach(({el, type, fn}) => el.removeEventListener(type, fn))
         }
+        this.redecorateAllInputs = () => {
```

Comment:
Triggered when we detect email protection sign in, so we can show the correct decoration.

Reply:
Note to self: shouldn't we rather call `this.execOnInputs(this.addInpt)`? This would follow the normal flow, including the right conditionals and would not create duplicates because we're using a `Set`.